logger/init.go

```diff
@@ -24,17 +24,12 @@ func GetFormatterWithForcedPrefix() *prefixed.TextFormatter {
 
 func GetLogger() *logrus.Logger {
 	level := os.Getenv("TYK_LOGLEVEL")
-	if level == "" {
```

Comment:
@mativm02 why is this removed?

Reply:
Because "info" is the default value. If the user sends an empty string, we don't need to convert it to "info", since it will join the "default" case in the switch sentence, which set the log level to "InfoLevel".
That's also the reason why I've removed `case "info":` in the switch sentence.

Reply:
makes sense!